front/lib/api/assistant/actions/retrieval.ts

```diff
@@ -469,7 +471,10 @@ const getRefs = () => {
       })
       .flat();
     // randomize
-    REFS.sort(() => Math.random() - 0.5);
+    REFS.sort(() => {
+      const r = getRand();
```

Comment:
Can we seed explicitly?

Reply:
It is above

Reply:
I meant with getRand(0) as an example?

Reply:
I'm not sure I understand -- what would be the goal?

Reply:
Rather getRand("REF_SEED") as per code :-)

The goal is to be explicit that we need seeding to avoid someone refactoring later. A comment can also serve the purpose of course

Reply:
Yes, but getRand is returned by a function called rand that is itself explicitly seeded with a static string. But you're right, this isn't super clear i'i fix 👍